### PR TITLE
[ADAM-1476] Treat `.` ALT allele as symbolic non-ref.

### LINKED ISSUE #1476: VCF line with `.` ALT gets dropped
Using [this VCF](https://github.com/HadoopGenomics/Hadoop-BAM/blob/master/src/test/resources/test.vcf), we drop:

```
20	1230237	.	T	.	47	PASS	NS=3;DP=13;AA=T	GT:GQ:DP:HQ	0|0:54:7:56,60	0|0:48:4:51,51	0/0:61:2
```

When using `sc.loadVariants`.

### PR DESCRIPTION
**Pull request by fnothaft**
Resolves #1476.

### CODE REVIEW

File: adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala [hunk 1]
```diff
@@ -182,31 +182,37 @@ class ADAMContextSuite extends ADAMFunSuite {
   sparkTest("can read a gzipped .vcf file") {
     val path = testFile("test.vcf.gz")
     val vcs = sc.loadVcf(path)
-    assert(vcs.rdd.count === 6)
+    assert(vcs.rdd.count === 7)
```
Comment (heuermh) [inline adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala]: were the compressed test resources changed too?

Comment (fnothaft) [inline adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala]: Nope... they appear to have been a compressed version of the VCF file from Hadoop-BAM.

Comment (heuermh) [inline adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala]: :facepalm:

**Comment by heuermh** [comment]
Thank you, @fnothaft!

**Comment by a1xt06** [comment]
Thank you!

**Pull request merged**